package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.25.1",
+  "version": "1.27.0-rc.2",
```

Comment:
We don't need to update this version for every release candidate.  We should be able to update it to 1.27.0 now.